### ISSUE TITLE
Mention the actual Caddy version on the Download page

### ISSUE THREAD

**Issue by anthonyvdotbe** [caddyserver/website#393]
The latest Caddy version is 2.8.4, but the Download page still offers version 2.8.0.
It would be helpful if the Download page would mention which Caddy version it currently offers.

**Comment by mholt** [comment]
Yeah that would be good, huh. Lemme see if I can figure that out... it kind of depends on if the build is already cached, if so it just uses that (but we should invalidate the cache with new Caddy versions, so that needs to happen before we can reliably show the version on the site).
